### PR TITLE
Occurrence of a number in array added

### PR DESCRIPTION
**Pull request by shivammahajancse**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#253 )

### CODE REVIEW

**Comment by shivammahajancse** [comment]
Please review sir @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786

File: docs/mathematics/occurrence-of-no.-in-array.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+title: Occurence of a number in array
+slug: /mathematics/occurrence-of-no.-in-array
+---
+
+## Introduction
+
+In this article we will see how to count the occurrence of a number in a given sets of data. Occurrence or frequency means how many times a number is present in a given sets of data.
+
+## Explanation 
+In this we have to count the number of times an given element is present in array that means if an element is present 4 times in array then we have to print 4.
```
Comment (iamrajiv) [inline docs/mathematics/occurrence-of-no.-in-array.mdx:11]: @shivammahajancse Update to this

```markdown
## Introduction

In this article, we will see how to count the occurrence of a number in a given set of data. Occurrence or frequency means how many times a number is present in a given set of data.

## Explanation

In this, we have to count the number of times a given element is present in an array that means if an element is present 4 times in the array then we have to print 4.
```

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

File: docs/mathematics/occurrence-of-no.-in-array.mdx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+```
+#### Output :
+
+```
+2
+```
+
+## Complexity Analysis
+ 
+- **Time Complexity:** `O(n)`
```
Comment (vybhav72954) [inline docs/mathematics/occurrence-of-no.-in-array.mdx:145]: Can we have the statement `where `n` is the number of elements.` after this line.

File: docs/mathematics/occurrence-of-no.-in-array.mdx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+- **Space Complexity:** `O(1)`
+
+where `n` is the number of elements.
```
Comment (vybhav72954) [inline docs/mathematics/occurrence-of-no.-in-array.mdx:149]: Credits?

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: docs/mathematics/occurrence-of-no.-in-array.mdx (hunk 3 above)
Comment (shivammahajancse) [reply docs/mathematics/occurrence-of-no.-in-array.mdx:149]: Sir code is written by me. Not present in neoalgo

**Comment by shivammahajancse** [comment]
All requested changes done sir @vybhav72954

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@shivammahajancse Please return count equal to zero if the given number is not present in the array. Also, do mention in the Explanation part

**Comment by shivammahajancse** [comment]
@siddharth25pandey sir all changes done

**Comment by shivammahajancse** [comment]
@atarax665 sir please review

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by shivammahajancse** [comment]
@ankitaggarwal23 sir merge conflict solved

**Comment by vybhav72954** [comment]
Dear @shivammahajancse I can't merge this PR as of now. NeoAlgo-Docs is the documentation of [NeoAlgo](https://github.com/TesseractCoding/NeoAlgo). Now, what to do? Simple.

Push the same code in NeoAAlgo **If** this script is already not present there. NeoAlgo has a really strict policy against plagiarism, we don't want to implement the same policy here as it would be a waste of time. So make a PR there, let others review it, get it merged and I will merge this as well.

**But** if the same script is already there, paste those scripts in your `.mdx`  file. Give the proper credits, and we are good to go.

Am closing this PR as of now. Kindly check NeoAlgo, see if there is a script or not, if not, make a PR there, get it merged, mention me in the comments here, and I will merge it instantly. If the script is there, prepare a new markdown file, mention me here, I will reopen the PR so that you can make the required changes, and then it can be merged.

**Comment by shivammahajancse** [comment]
114140237-a2de7c80-992d-11eb-94a6-7eb13dd09ef0
@vybhav72954 sir here it is written we can either write our own code sir

**Review by siddharth25pandey — APPROVED** [review]
@shivammahajancse LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @shivammahajancse for his code

**Pull request merged**